test/jdk/java/awt/print/PrinterJob/ImagePrinting/AlphaPrintingOffsets.java

```diff
@@ -0,0 +1,237 @@
+                    "to tested bug affects pages printed with transparency " +
+                    "in LANDSCAPE and REVERSE_LANDSCAPE orientations there " +
+                    "is an option to print only 2 pages affected. " +
+                    "To do it pass PaperSavingMode parameter.";
+
+    private static boolean isPaperSavingMode = false;
+
+    public static void main(String[] args) throws Exception {
+        if (PrinterJob.lookupPrintServices().length > 0) {
+
+            String instructionsHeader = "This test prints 6 pages with page " +
+                    "margin rectangle and a text message. \n";
+            if (args.length > 0) {
+                isPaperSavingMode = args[0].equals("PaperSavingMode");
+            }
+            if (isPaperSavingMode) {
+                instructionsHeader = "This test prints 2 pages with page " +
+                        "margin rectangle and a text message. \n";
+            }
+            PassFailJFrame.builder().instructions(instructionsHeader + INSTRUCTIONS)
+                    .rows(15).testUI(() -> createTestUI()).build().awaitAndCheck();
+
+        } else {
+            System.out.println("Test failed : " +
+                    "Printer not configured or available.");
+            throw new RuntimeException("Test failed : " +
+                    "Printer not configured or available.");
+        }
+
+    }
+
+    public static JFrame createTestUI() {
+        JFrame testUI = new JFrame("Print images");
+        testUI.setSize(1, 1);
+        testUI.addWindowListener(new WindowAdapter() {
+            @Override
+            public void windowOpened(WindowEvent e) {
+                super.windowOpened(e);
+                print();
+            }
+
+            @Override
+            public void windowActivated(WindowEvent e) {
+                super.windowActivated(e);
+                testUI.setVisible(false);
+            }
+        });
+
+        return testUI;
+    }
+
+    private static void print() {
+        PrinterJob printerJob = PrinterJob.getPrinterJob();
+        PageFormat pageFormatP = printerJob.defaultPage();
+
+        Paper paper = pageFormatP.getPaper();
+        paper.setImageableArea(0, 0, paper.getWidth(), paper.getHeight());
+        pageFormatP.setPaper(paper);
+
+        PageFormat pageFormatL = (PageFormat) pageFormatP.clone();
+        PageFormat pageFormatRL = (PageFormat) pageFormatP.clone();
+
+        pageFormatL.setOrientation(LANDSCAPE);
+        pageFormatRL.setOrientation(REVERSE_LANDSCAPE);
+
+        Printable printableTransparent = new CustomPrintable(254);
+        if (isPaperSavingMode) {
+            Book bookPageSavingTest = new Book();
+            bookPageSavingTest.append(printableTransparent, pageFormatL);
+            bookPageSavingTest.append(printableTransparent, pageFormatRL);
+            printerJob.setPageable(bookPageSavingTest);
+        }
+        else {
```

Comment:
```suggestion
        } else {
```
This is the Java style and you follow this style in other places.